lib/ts/postSuperTokensInitCallbacks.ts

```diff
@@ -12,11 +12,17 @@
  * License for the specific language governing permissions and limitations
  * under the License.
  */
-import { normaliseAuthRecipe } from "../authRecipe/utils";
-import { InputType, NormalisedInputType } from "./types";
 
-export function normaliseAuthRecipeWithEmailVerificationConfig<Action>(
-    config: InputType<Action>
-): NormalisedInputType<Action> {
-    return normaliseAuthRecipe(config);
+export class PostSuperTokensInitCallbacks {
```

Comment:
Will this also be required in auth-react repo? Cause in that repo, we do not call web-js' init function, so this will not be called after auth-react.init -> does this mean that the validators won't be added? That would be a problem right?

One way to solve this would be to expose this from the web-js SDK and call the `runPostInitCallbacks` in auth-react as well.

Reply:
If we have to expose this, we can do that via `supertokens-web-js/utils/postSuperTokensInitCallbacks`